app/components/market/submit/submit-market-application.html

```diff
@@ -39,9 +39,9 @@
             </div>
         </bl-form-page>
     </bl-complex-form>
-    <div class="submit-box" *ngIf="modeState !== Modes.None">
+    <div [title]="getToolTip()" class="submit-box" *ngIf="modeState !== Modes.None">
         <bl-server-error [error]="error"> </bl-server-error>
-        <bl-button [action]="submit" type="wide" class="submit-job" color="success">
+        <bl-button [disabled]="!isFormValid()" [action]="submit" type="wide" class="submit-job" color="success">
```

Comment:
make the tooltip here instead of being on the big form

Reply:
done